venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_tabs.js

```diff
@@ -125,6 +136,10 @@ function renderConferenceTabs() {
       id: 'my-submitted-papers',
     },
     {
+      heading: 'My Original Papers',
```

Comment:
So, what is the difference between 'My Submitted Papers' and 'My Original Papers'?  One displays the blinded notes other the original notes?

For me is enough showing only one tab(Submitted Papers) and telling to the user that they can access to the original through the 'Modifiable Original' link or Task view. (my opinion, I may be wrong)

Reply:
yes, one shows blinded, the other shows originals.

yeah, I'm not sure either. but my thought is that showing both these tabs next to each other (with these instructions right above) will prompt the user to understand what's really happening sooner in the submission process (i.e. as soon as they submit a paper, they'll see that there is both an original and a copy).

it also means that the user doesn't have to go to the tasks page as often, which I think is a good thing (since we want to redesign it soon). 

but yes, I agree that it's not obviously the right solution. maybe zach has an opinion. alternatively we could show it to the program chairs on the dev site and see what they think

Reply:
Both tabs are going to show the same data with a slight difference: the author names, and in most of the cases is going to be one only paper. I think is redundant to show both and also I wouldn't like to display them separately, because they are the same thing, they represent the same paper, they are linked.

Reply:
I agree with Melisa that it seems redundant and a little confusing at first. Can we do what we do in other places and have some indication that the author name is only visible to you? ("privately revealed to you").

The other alternative I can see is removing the My Submissions tab and just keeping My Originals

Reply:
I don't know if adding that text would help the problem I'm trying to solve here, which is that users don't have a good idea about how the original/copy system works. But maybe the instructional text is enough.

If we're going to keep one of these tabs, I do think it needs to be the "My Submissions" tab - I think easy access to the discussion of the papers (which is only present in the blinded notes) is more important than easy access to the originals.

I'll take out the my originals tab and fix the text.

Reply:
I agree with that, also you can add a tooltip in the 'Modifiable Original' button with some help text. @zbialecki can help you to do that.

Reply:
Adding a tooltip to a button is easy - you just have to add a little markup to the html and call a JS function after the button is added to the page.
https://getbootstrap.com/docs/3.3/javascript/#tooltips